content/en/tracing/guide/trace_queries_dataset.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How Trace Queries Source Data
+kind: guide
+further_reading:
+    - link: "/tracing/trace_explorer/trace_queries/"
+      tag: "Documentation"
+      text: "Trace Queries"
+---
+
+## Overview
+ 
+With Trace Queries, you can find entire traces based on the properties of multiple spans and the relationships between those spans within the structure of the trace. To learn more, read the [Trace Queries documentation][1].
+
+{{< img src="tracing/trace_queries/trace_queries.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Queries UI" >}}
+
+## How Trace Queries source data
+
+Datadog uses the [Intelligent Retention Filter][6] to index data for Trace Queries. It does so by performing: 
+
+- [Flat sampling](#1-flat-sampling): A uniform 1% sample of ingested spans.
+- [Diversity sampling](#diversity-sampling): A representative, diverse selection of traces to keep visibility over each environment, service, operation, and resource.
+
+{{< img src="tracing/trace_queries/trace_queries_new_dataset.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="1% Flat Sampling & Diversity Sampling" >}}
+
+**Note**: Spans indexed by flat sampling and diversity sampling do not count towards the usage of indexed spans, and therefore, **do not impact your bill**.
+
+### 1% flat sampling
+
+Flat 1% sampling is applied based on the `trace_id`, meaning that all spans belonging to the same trace share the same sampling decision. To learn more, read the [one percent flat sampling documentation][2].
```

Comment:
Maybe this is outside the scope of what we want to consider here, but I think we should look into expanding the 1% flat sampling documentation. There isn't really any extra info in the section we link to compared to the overview here.

Reply:
Yep, I'll enrich the 1% flat sampling section